app/views/users/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class="show">
+  <%= render 'profile', user: @user %>
+  <div class="post-wrapper">
+    <% if @user == current_user %>
+      <%= render 'posts/form', post: Post.new %>
+    <% end %>
+    <ul>
+      <% @posts.each do |post| %>

```

Comment:
Please refer to the rails guides on a better way to render a partial for a collection.